client/src/components/Feed/Post.js

```diff
@@ -183,6 +183,35 @@ const Post = ({
     }
   };
 
+  const deleteComment = async (comment) => {
+    let response;
+    const postId = comment.postId;
+    const commentId = comment._id;
+    if (isAuthenticated && comment.author.id === user.id) {
+      const endPoint = `/api/posts/${postId}/comments/${commentId}`;
+
+      try {
+        response = await axios.delete(endPoint);
+      } catch (error) {
+        console.log({ error });
+      }
+      if (response && response.data) {
```

Comment:
ok so as far as I understand this is not like what I previously explained about "eventual consistency" but it only updated the UI after the request returned successfully
this is not really a problem, it might look less responsive in the UI eventually though, I think it generally looks much faster if the UI is updated first while the request is still in progress and then rolled back if something went wrong

Reply:
I'm going to merge maybe we should open another ticket for eventual consistency

Reply:
Or if you're all right approve and merge then we can move stuff to other ticket ([delete], eventual consistency, etc).